### PR TITLE
Added onnx export support for several multiclass classifiers

### PR DESCRIPTION
**Pull request by harishsk**
This PR adds Onnx export support + tests for the following multiclass classifiers:
* LbfgsMaximumEntropyMulticlassTrainer
* LightGbmMulticlassTrainer
* LightGbmMulticlassTrainer with SoftMax
* OneVersusAllTrainer
* SdcaMaximumEntropyMulticlassTrainer
* SdcaNonCalibratedMulticlassTrainer

There is a question marked with a REVIEW comment in the PR below. Please comment if you know of a better way to do what is done there.

### CODE REVIEW

File: src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs [hunk 1]
```diff
@@ -499,6 +505,10 @@ private abstract class ImplBase : ISingleCanSavePfa
             public abstract ValueMapper<VBuffer<float>, VBuffer<float>> GetMapper();
             public abstract JToken SaveAsPfa(BoundPfaContext ctx, JToken input);
 
+            public bool CanSaveOnnx(OnnxContext ctx) => Predictors.All(pred => (pred as ICanSaveOnnx)?.CanSaveOnnx(ctx) == true);
```
Comment (ganik) [inline src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs]: >(pred as ICanSaveOnnx)? [](start = 79, length = 23)

What happens if it cant cast?

Comment (harishsk) [reply src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs]: If it can't cast, it will return a null. And the question mark after the ')' ensures makes it a null conditional operator and will not execute further and the overall function will return false.

---
In reply to: [348211703](https://github.com/dotnet/machinelearning/pull/4462#discussion_r348211703) [](ancestors = 348211703)
